src/telemetry/stats.h

```diff
@@ -46,13 +47,13 @@ typedef struct BaseStats
 
 typedef struct StorageStats
 {
-	BaseStats base;
+	BaseStats bs;
```

Comment:
This renaming makes the code unnecessarily hard to read. The original name was better.